examples/advanced/edge/README.md

```diff
@@ -0,0 +1,66 @@
+# Running Edge Example with Hierarchical Clients
+
+Please follow these steps to run the Edge device emulator,
+
+## Provision
+
+Use tree_prov.py to generate a hierarchical NVFlare system with 2 levels and 2 clients at each level,
+
+     python tree_prov.py -p /tmp/edge_example -d 2 -w 2
```

Comment:
shall we mention full path here nvflare/edge/tree_prov.py? and for this exp, it will be --depth 1 --width 2